Documentation/README_Tooltip.md

```diff
@@ -1,39 +1,27 @@
+The [example scene](https://gfycat.com/WarmOblongBilby) demonstrates two ways to display a Tooltip on an object.
 
-- Add a GameObject and a **ToolTip prefab** object to the Scene Hierarchy.
-- In the ToolTip prefab's Inspector panel, expand the Tool Tip (Script). Select a TipState and set other settings. 
-- Enter the ToolTip text in the Text field. 
-- Expand the ToolTipConnector (Script). Drag the object that is to have the ToolTip from the Hierarchy into the field labelled Target. This attaches the ToolToolTip connector to the object. 
+## How to use a tool tip ##
+A tool tip can be added directly to the hierarchy and targeted to an object.
 
-This use of ToolTip assumes a ToolTip that is always showing or that is shown/hid in script by changing the TipState property of the ToolTip component.
+To use this method simply add a game object and one of the [tool tip prefabs](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/Assets/MixedRealityToolkit.SDK/Features/UX/Prefabs/Tooltips) to the scene hierarchy. In the prefab's inspector panel, expand the *Tool Tip* (script). Select a tip state and configure the tool tip.  Enter the respective text for the tool tip in the text field. Expand the *ToolTipConnector* (Script) and drag the object that is to have the tool tip from the hierarchy into the field labelled *Target*. This attaches the tool tip to the object. 
 
+This use assumes a tool tip that is always showing or that is shown / hidden via script by changing the tip state property of the tool tip component.
  
-## Dynamically spawning
-A ToolTip can be dynamically added to an object at runtime as well as pre-set to show and hide on a Tap or focus. Simply add the **ToolTipSpawner** script to any GameObject. In the script's Inspector, you can set delays for appearing and disappearing. You can also set a lifetime so that the ToolTip when spawned, will disappear after a duration. You can also set style properties such as Background in the ToolTipSpawner script. By default the ToolTip will be anchored to the object with the ToolTipSpawner script. You can override this by assigning a GameObject to the Anchor field.
+## Dynamically spawning tool tips ##
+A tool tip can be dynamically added to an object at runtime as well as pre-set to show and hide on a tap or focus. Simply add the [`ToolTipSpawner`](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Tooltips/ToolTipSpawner.cs) script to any game object. Delays for appearing and disappearing can be set in the scripts inspector as well as a lifetime so that the tool tip will disappear after a set duration. Tool tips also feature style properties such as background visuals in the spawner script. By default the tool tip will be anchored to the object with the spawner script. This can be changed by assigning a game object to the anchor field.
```

Comment:
```suggestion
A tooltip can be dynamically added to an object at runtime as well as pre-set to show and hide on a tap or focus. Simply add the [`ToolTipSpawner`](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Tooltips/ToolTipSpawner.cs) script to any game object. Delays for appearing and disappearing can be set in the scripts inspector as well as a lifetime so that the tooltip will disappear after a set duration. Tooltips also feature style properties such as background visuals in the spawner script. By default the tooltip will be anchored to the object with the spawner script. This can be changed by assigning a GameObject to the anchor field.
```